prompts.py

```diff
@@ -20,17 +20,16 @@
 """
 
 TITLE_GENERATION_PROMPT = """
-Based on the following information, generate a catchy and appropriate title for a song:
+Based on the following lyrics and style, generate a song title in {language}.
 
 Original title: {title}
 Lyrics: {lyrics}[:200]...
-Style: {style}
-Language: {language}
-Songwriter's thought: {thought}
+Style:
+{style}
 
-Please provide only the generated title in the specified language, without any additional explanation.
-If the language is Chinese, use Traditional Chinese characters.
```

Comment:
Traditional Chinese is not the same as the Simplified Chinese, so we need to specify it.  (Or we need add more selection options at app.py)